<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBFullScheduleView</class>
 <widget class="QWidget" name="DBFullScheduleView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblMode">
       <property name="text">
        <string>Modo de visualización</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbMode">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <item>
        <property name="text">
         <string>Vista del día</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vista semanal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vista mensual</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblDate">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbAdjust">
       <property name="text">
        <string>Ajustar visualización a espacio visible</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/generales/images/edit_resizelines.png</normaloff>:/generales/images/edit_resizelines.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPrevious">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/actions/actions/previous.png</normaloff>:/actions/actions/previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNext">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/actions/actions/next.png</normaloff>:/actions/actions/next.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DBScheduleView" name="dbScheduleView">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DBScheduleView</class>
   <extends>QFrame</extends>
   <header>widgets/dbscheduleview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbMode</tabstop>
  <tabstop>pbPrevious</tabstop>
  <tabstop>pbNext</tabstop>
  <tabstop>dbScheduleView</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
